bin/cron/dms_tasks

```diff
@@ -29,11 +31,29 @@ def get_delay(task_execution_time_sec)
   return (task_execution_time_sec * 1.3 + 300).to_i
 end
 
+# @return [nil | String] The channel ID for the data room in Slack. Returns nil
+#   if not found.
+def get_data_channel_id
+  # Documentation at https://api.slack.com/methods/channels.list.
+  slack_api_url = "https://slack.com/api/channels.list?token=#{SLACK_TOKEN}"
+  channels = open(slack_api_url).read
+  parsed_channels = JSON.parse(channels)
+  parsed_channels['channels'].each do |parsed_channel|
+    return parsed_channel['id'] if parsed_channel['name'] == 'data'
+  end
+  return nil
+end
```

Comment:
Note that it is also possible to simply store the `data` room channel ID as a CDO secret, as done in `update_dotd` for the `developers` room. Let me know if that is preferred.